chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -409,3 +409,21 @@
   group "root"
   mode "0644"
 end
+
+crowbar_node = node_search_with_cache("roles:crowbar").first
+address = crowbar_node["crowbar"]["network"]["admin"]["address"]
+protocol = crowbar_node["crowbar"]["apache"]["ssl"] ? "https" : "http"
+server = "#{protocol}://#{address}"
+password = crowbar_node["crowbar"]["users"]["crowbar"]["password"]
+verify_ssl = !crowbar_node["crowbar"]["apache"]["insecure"]
+template "/etc/crowbarrc" do
+  source "crowbarrc.erb"
+  variables(
+    server: server,
+    password: password,
+    verify_ssl: verify_ssl
+  )
+  owner "root"
+  group "root"
+  mode "0o644"
```

Comment:
Please don't make it world-readable as it contains a password.

Reply:
Wow I can't believe I did that

Reply:
fixed